pkgs/by-name/ch/chirp/package.nix

```diff
@@ -3,12 +3,12 @@
 , writeShellScript
 , glib
 , gsettings-desktop-schemas
-, python3
+, python311
```

Comment:
what is the reason for this change? Do we pin python applications to specific python versions now?

Reply:
I guess it was some issue with 3.12, but it looks ok now. Well, removed pinning.